dashboard/app/helpers/users_helper.rb

```diff
@@ -5,6 +5,22 @@ module UsersHelper
   include ApplicationHelper
   include SharedConstants
 
+  def check_and_apply_clever_takeover(user)
```

Comment:
Should this method check `cookies['pm'] == 'clever_takeover'` and raise if it's reached without the flag, just in case?

Reply:
:+1: